src/uct/ib/ud/base/ud_ep.h

```diff
@@ -249,6 +249,7 @@ struct uct_ud_ep {
     uint16_t         flags;
     uint8_t          path_bits;
     ucs_wtimer_t     slow_timer;
+    ucs_wtimer_t     destroy_timer;
```

Comment:
can we reuse the slow_timer?

Reply:
The EP should respond some time to pass some tests. As far as I understand, only 1 callback can be associated with the timer.

Reply:
maybe there are other fields which can be reused? we know the user will not send on the ep anymore. maybe the tx window?

Reply:
tx window is not empty here at least because of scheduling of flush in the end, also EP should send ACKs

Reply:
I'll try to find something else

Reply:
Seems like EP should not lose functionality. Playing with the fields placement I can reduce it by 8 bytes only, it's too small to reduce cache lines number

Reply:
maybe we can set the destroy timer when slow_timer runs out and not needed any more?

Reply:
yes, timer is not fast path